arm/Microsoft.Compute/virtualMachines/readme.md

```diff
@@ -559,3 +560,34 @@ You can specify multiple user assigned identities to a resource by providing add
 - [Vaults/Backupfabrics/Protectioncontainers/Protecteditems](https://docs.microsoft.com/en-us/azure/templates/Microsoft.RecoveryServices/2021-06-01/vaults/backupFabrics/protectionContainers/protectedItems)
 - [Virtualmachines](https://docs.microsoft.com/en-us/azure/templates/Microsoft.Compute/2021-07-01/virtualMachines)
 - [Virtualmachines/Extensions](https://docs.microsoft.com/en-us/azure/templates/Microsoft.Compute/2021-07-01/virtualMachines/extensions)
+
+## Usage examples
+
+### Usage example 1
+
+```json
+```
+
+```bicep
+```
+### Usage example 2
+
+```json
+```
+
+```bicep
+```
+### Usage example 3
+
+```json
+```
+
+```bicep
+```
+### Usage example 4
+
+```json
+```
+
+```bicep
+```
```

Comment:
mumble mumble

Reply:
Looks like I had the wrong function still loaded to memery. Fixed @eriqua